app/components/UI/FiatOnRampAggregator/components/TransactionDetails.tsx

```diff
@@ -244,12 +277,12 @@ const TransactionDetails: React.FC<Props> = ({ order }: Props) => {
 					<ListItem.Amount>
 						<Text small bold primary>
 							{order.data.fiatCurrency.denomSymbol}
-							{order.amount} {order.data.fiatCurrency.symbol}
+							{renderFiat(order.amount, order.data.fiatCurrency.symbol, order.data.fiatCurrency.decimals)}
 						</Text>
 					</ListItem.Amount>
 				</ListItem.Content>
 				{order.state === SDK_ORDER_STATUS.Completed && (
-					<TouchableOpacity>
+					<TouchableOpacity onPress={() => handleLinkPress('https://etherscan.io/tx/' + order.data.txHash)}>
```

Comment:
This is incorrect since these orders are from different networks. 

I'm trying to figure out how to solve this but we should definitely use `import etherscanLink from '@metamask/etherscan-link';`